proposals/text/README.md

```diff
@@ -165,6 +165,7 @@ The primvars:displaycolor and the primvars:displayopacity will be the default co
 - markupLanguage. It is a string token tells how the markup tags are interpreted. Currently we support "noMarkup" which is for plain text string, and "mtext" which is for MTEXT markups. By default it is "noMarkup".
 - primvars:backgroundColor. The color of the background. By default there is no background color. It can be overridden by the specified background color in the markup. (Currently this not supported in our implementation. Will be implemented in the future.)
 - primvars:backgroundOpacity. The opacity of the background. By default there is no background opacity. The background is completely transparent. It can be overridden by the specified background color in the markup. (Currently this not supported in our implementation. Will be implemented in the future.)
+- textMetricsUnit. The unit for textHeight and other text metrics. It is a string token, which could be "pixel", "publishingPoint" or "worldUnit". By default it is "worldUnit". If the text primitive doesn't have billboard, only "worldUnit" is valid. If the unit is set to other value, we will still use "worldUnit" unit. The unit of the textHeight will be the same as the unit of the world space. If the text primitive has billboard, that it always faces to the camera and the size will not change, only the value "pixel" and "publishingPoint" are effective. If metrics unit is "pixel", the unit will be pixel on the screen. If metrics unit is "publishingPoint", the unit will be desktop publishing point or 1/72 inch on the screen. If the unit is set to "worldUnit", we will use "pixel" as the value. 
```

Comment:
same comment as above. I realize this text is pre-existing, but on close read, I'm finding it difficult to understand, unfortunately.